src/lib/snykGroup.ts

```diff
@@ -43,7 +49,12 @@ export class snykGroup {
     this._requestManager = new requestsManager({
       snykToken: this.key,
       userAgentPrefix: 'snyk-user-sync-tool',
+      burstSize: 1,
+      maxRetryCount: 20,
```

Comment:
I was being particularly conservative to avoid a scenario where someone is assigned to a collaborator role but then not reconfigured to a custom role because the max retry is hit (since 2 api calls are required to set a custom role). I think I'll change this to 10 and test on a small set of role changes (so more faster retry’s) and make sure it works okay